### ISSUE TITLE
Huboard specific labels do not have historical tracking

### ISSUE THREAD

**Issue by benjamincharity** [huboard/huboard#659]
#### Description:

One of the major benefits of Github issues (imo) is the historical timeline. eg `ohill133 added the 2 - Needs Discussion label 17 hours ago`. Because the Huboard `Mark as blocked` is so available, it's hard for my team members to not use it. But if they _do_ use and remove it there is no way to know the label ever existed. When was it unblocked? Who unblocked it?

#### Steps to reproduce:

- Create ticket
- Mark ticket as blocked within Huboard
- Unmark ticket as blocked.
- No way to know if it was ever blocked and/or who blocked/unblocked it.

(Note: I thought this was brought up elsewhere (possibly as part of another discussion) but I could not find it... so sorry if a duplicate!)

**Comment by rauhryan** [comment]
Hey @benjamincharity, 

There is a related open issue #399 

We ran into some concerns about using a blocked label as the source of truth for the blocked status. 

Now that we've shipped a better syncing mechanism we should be able to revisit this issue.

We have some bigger engineering efforts in flight right now, I've added this to the our polish list so we'll try to sneak it in soon.

**Comment by benjamincharity** [comment]
Ahh I knew there was one out there! I will move my original post here to a comment there and close this out as a duplicate.

**Comment by benjamincharity** [comment]
Closing as duplicate of https://github.com/huboard/huboard/issues/399

**Comment by rauhryan** [comment]
Thanks!
